packages/react-ui-core/src/Menu/Menu.js

```diff
@@ -116,7 +132,7 @@ export default class Menu extends PureComponent {
           className,
           theme.Menu
         )}
-        items={options}
+        items={this.optionLabels}
```

Comment:
Seems a little specific to call this`optionLabels` doesn't it?  How about we just combine the options getter and make one getter?  We're only seem to be referencing `this.options` in one place.

Reply:
Actually there are a couple other places that use options that should use the options getter, so rather than combine these, I'm going to actually extend the use of the options getter.